src/auth/authExplorerProvider.ts

```diff
@@ -22,16 +23,16 @@ export class AuthExplorerProvider implements ExplorerProvider {
     return [
       new ExplorerTreeItem({
         resourceType: ExplorerResourceTypes.signup,
-        label: l10n.t('Create an Account with Affinidi'),
+        label: `${labels.createAnAccountWithAffinidi}`,
```

Comment:
Are backticks necessary? Couldn't we do `label: labels.createAnAccountWithAffinidi,`
This repeats all over the MR.

Reply:
removed